docker/develop/x86_64/Dockerfile

```diff
@@ -186,6 +186,12 @@ RUN git clone git://github.com/hyperledger/iroha-ed25519.git /tmp/ed25519; \
     ldconfig; \
     rm -rf /tmp/ed25519
 
+# fetch lcov reports converter
+RUN curl -L -o /tmp/lcov_cobertura.py https://raw.github.com/eriwen/lcov-to-cobertura-xml/master/lcov_cobertura/lcov_cobertura.py
```

Comment:
Looks dangerous

Reply:
You mean it's better to download by explicitly stating the commit number?

Reply:
Exactly, this file can be removed/modified in malicious way

Reply:
@l4l it is a develop docker, nobody cares